app/controllers/api/v0/api_controller.rb

```diff
@@ -1,4 +1,11 @@
-class Api::V0::ApiController < ApplicationController
+class Api::V0::ApiController < ActionController::Base
+  protect_from_forgery with: :exception, prepend: true
+
+  include ValidRequest
```

Comment:
see https://github.com/thepracticaldev/dev.to/blob/master/app/controllers/concerns/valid_request.rb - added this back in as it's on for all `ApplicationController` actions